<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaskDryOptionsBasic</class>
 <widget class="QDialog" name="MaskDryOptionsBasic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>213</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="maskDryTitle">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>16</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#BDBDBD</string>
   </property>
   <property name="text">
    <string>Mask Dry</string>
   </property>
  </widget>
  <widget class="QPushButton" name="maskDryCancel">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>170</y>
     <width>343</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color:#3B3B3B;
color:#BDBDBD;
border:none;
border-radius:3px;
}
QPushButton:pressed {
background-color:#606060;
color:#BDBDBD;
border:none;
border-radius:3px;
}
</string>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="maskDryTestStartStop">
   <property name="geometry">
    <rect>
     <x>96</x>
     <y>103</y>
     <width>158</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color:#3B3B3B;
color:#BDBDBD;
border:none;
border-radius:3px;
}
QPushButton:pressed {
background-color:#606060;
color:#BDBDBD;
border:none;
border-radius:3px;
}
</string>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QWidget" name="spinnerParent" native="true">
   <property name="geometry">
    <rect>
     <x>145</x>
     <y>30</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>349</width>
     <height>213</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(49, 49, 49);
border: 1px;
border-radius: 5px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QProgressBar" name="maskDryProgressBar">
   <property name="geometry">
    <rect>
     <x>57</x>
     <y>65</y>
     <width>244</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar {
    border: 1px solid grey;
    border-radius: 5px;
    text-align: centre;
}
QProgressBar::chunk {
    background-color: #1788c3;
    width: 20px;
    text-align: centre;
}
</string>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="textVisible">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>maskDryTitle</zorder>
  <zorder>maskDryCancel</zorder>
  <zorder>maskDryTestStartStop</zorder>
  <zorder>spinnerParent</zorder>
  <zorder>maskDryProgressBar</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
